package.json

```diff
@@ -13,12 +13,15 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "chokidar-cli": "3.0.0",
+    "concurrently": "8.2.2",
     "eslint": "^8",
     "eslint-config-next": "14.0.0",
-    "prettier": "^3.0.3",
-    "typescript": "^5",
     "lerna": "7.4.2",
-    "chokidar-cli": "3.0.0",
-    "concurrently": "8.2.2"
+    "prettier": "^3.0.3",
+    "typescript": "^5"
+  },
+  "dependencies": {
```

Comment:
If we need this as a part of the package - it should live in the nook package.json